<?xml version="1.0" ?>
<testsuite errors="1" failures="0" name="TestDowntimes-20190504152128" tests="5" time="0.164">
	<testcase classname="TestDowntimes" name="test_downtimesfile" time="0.081"/>
	<testcase classname="TestDowntimes" name="test_setdelayedperiod" time="0.009"/>
	<testcase classname="TestDowntimes" name="test_setfrontendsecclass" time="0.017"/>
	<testcase classname="TestDowntimes" name="test_setperiodwithendtime" time="0.007"/>
	<testcase classname="TestDowntimes" name="test_entryonlydowntime" time="0.049">
		<error message="True is not false" type="AssertionError">
<![CDATA[Traceback (most recent call last):
  File "./test_glideFactoryDowntimeLib.py", line 118, in test_entryonlydowntime
    check_time=now + 1))
AssertionError: True is not false
]]>		</error>
	</testcase>
	<system-out>
<![CDATA[]]>	</system-out>
	<system-err>
<![CDATA[]]>	</system-err>
</testsuite>
